workflow/tests/hpxml_translator_test.rb

```diff
@@ -230,6 +230,24 @@ def test_run_simulation_timeseries_output_variables
     assert_equal(1, timeseries_rows[0].select { |r| r == 'Zone People Total Heating Energy: Living Space' }.size)
   end
 
+  def test_run_simulation_utility_bills
```

Comment:
Can you also use `--add-utility-bills` in the `_run_xml` method so that it gets used for all workflow sample file runs? And make sure these utility bill results get captured in the CI artifacts/comparisons, either as new columns in results.csv or a new results_bills.csv (similar to the existing results_hvac_sizing.csv).

Reply:
Now adding `--add-utility-bills` for all sample xml files except for `base-location-capetown-zaf.xml` (no rates and so simulation errors out).

A `results_bills.csv` now gets exported to `workflow/tests/base_results` for all other xml files (excluding those with DSE).

Reply:
Can we change it so that this measure never "errors out"? If we can't calculate bills, don't produce an output file, but return true to indicate that the measure/workflow was successful (like we do for DSE). I don't want users to not be able to get simulation results just because utility bills couldn't be calculated.